test/unit/prof_threshold.c

```diff
@@ -0,0 +1,103 @@
+#include "test/jemalloc_test.h"
+
+// Test config (set in reset_test_config)
+const int ALLOC_ITERATIONS_IN_THRESHOLD = 10;
```

Comment:
nit: Just to make it a bit more C-style.

```suggestion
#define ALLOC_ITERATIONS_IN_THRESHOLD 10;
```